packages/mobile/src/components/core/LockedStatusBadge.tsx

```diff
@@ -10,6 +10,7 @@ const useStyles = makeStyles(({ palette, spacing, typography }) => ({
   root: {
     backgroundColor: palette.accentBlue,
     paddingHorizontal: spacing(2),
+    paddingVertical: 1,
```

Comment:
If I'm reading the mocks correctly, this should be 2 pixels (or a half-unit)?
![image](https://github.com/AudiusProject/audius-client/assets/1815175/72629f66-0ba7-4b42-9a14-9ab21d642b66)

Reply:
Yeah that confused me for a sec, but in the mock it also sets height to 16px which limits the padding to 1: icon is 14px + 1px on each side.